.golangci.yml

```diff
@@ -34,7 +34,6 @@ linters:
         - unconvert
         - unconvert
         - unparam
-        - whitespace
```

Comment:
Why removing some linters?

Reply:
We don't use this one in the mono repo, and it was complaining about some code structure that imo is totally fine.

Reply:
Because our mono-repo is huge and lots of linters we don't have there. SDK and Terraform are the most strict ones, which are also "boilerplates" for new repos

Reply:
Okay, but then we need to tweak the settings a bit. Look at this.

In the PR currently, in `ingest_record.go`, I have this code
![image](https://user-images.githubusercontent.com/11062293/235624961-7499002c-6196-4b93-a32b-1c3e0d124e44.png)

to which the linter complains about unnecessary whitespaces 
![image](https://user-images.githubusercontent.com/11062293/235625036-ad7dd9e4-79bd-45bf-ad4d-3e99db76f543.png)

So if I remove them, the `whitespace` linter tool is happy, but for it to be happy, the line is so long that the `lll` tool starts complaining (and rightfully so)

![image](https://user-images.githubusercontent.com/11062293/235625310-ec380a63-5f99-485f-a912-79ee55b2f587.png)
![image](https://user-images.githubusercontent.com/11062293/235625348-8c7cded6-7ab3-4eae-a4ba-a709174a245b.png)

Reply:
It complains that you have empty line after `{`, not that you have to make declaration one-liner

Reply:
I hereby concede 😄 Adding it back